### DISCUSSION TITLE
JDK 17 support for linux machine

### DISCUSSION THREAD

**Discussion by VadimZhuk0v** [codemagic-ci-cd/codemagic-docs#1590] [Q&A]
Google recently moved to gradle plugin 8.0.0 where JDK 17 is required. Currently we use linux machine and we want to keep going with linux. Could you add support for that JDK version?

**Comment by mohammedbabelly20** [comment]
Hey @VadimZhuk0v, this should be available on our next update, hopefully by next week.
